packages/stack/src/styles/Stack.module.css

```diff
@@ -0,0 +1,60 @@
+/* stylelint-disable-next-line */
+.stack {
+}
+
+.item:last-child {
+  padding-bottom: var(--lp-spacing-100);
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+}
+
+.left {
+  align-items: flex-start;
+}
+
+.center {
+  align-items: center;
+}
+
+.right {
+  align-items: flex-end;
+}
+
+.gap-0 {
```

Comment:
I know this is the first pass, but while we're at it, what do you think about switching to `gap` instead of padding? That's really what we want here.